shared/constants/network.ts

```diff
@@ -207,6 +208,7 @@ export const CHAINLIST_CHAIN_IDS_MAP = {
   ZORA_MAINNET: '0x76adf1',
   FILECOIN: '0x13a',
   NUMBERS: '0x290b',
+  APE: '0x8173',
```

Comment:
This PR is fine since it's a cherry pick.  But `CHAINLIST_CHAIN_IDS_MAP` already includes the spread operator `...CHAIN_IDS` so seems like it would already get the `APE_MAINNET` entry from there?

Although I don't really know the difference between `CHAIN_IDS` and `CHAINLIST_CHAIN_IDS_MAP` anyway.